nixos/doc/manual/installation/installing.chapter.md

```diff
@@ -1,30 +1,147 @@
+     The key to open the boot menu is different across computer brands and even
+     models. It is usually <kbd>F12</kbd>, but sometimes <kbd>Esc</kbd> or
+     another function key. If you are unsure, look at the POST screen that is
+     shown first when the computer is started. You can also search online for
+     your computers brand, model followed by "boot from usb".
+     The computer might not even have that feature, so you have to go into the
+     BIOS/UEFI settings to change the boot order. Again, search online for
+     details about your specific computer model.
+
+     For Apple computers with Intel processor press and hold the <kbd>⌥</kbd>
+     (Option or Alt) key until you see the boot menu. On Apple silicon press
+     and hold the power button.
+     :::
+
+     ::: {.note}
+     If your computer support both BIOS and UEFI boot, choose the UEFI option.
+     :::
+
+     ::: {.note}
+     If you use a CD for the installation, the computer will probably boot from
+     it automatically. If not, choose the option containing the word "CD" from
+     the boot menu.
+     :::
+
+3.   Shortly after selecting the appropriate boot drive, you should be
+     presented with a menu with different installer options. Leave the default
+     and wait (or press <kbd>Enter</kbd> to speed up).
+
+4.   The graphical images will start their corresponding desktop environment
+     and the graphical installer, which can take some time. The minimal images
+     will boot to a command line. You have to follow the instructions in
+     [](#sec-installation-manual) there.
+
+## Graphical Installation {#sec-installation-graphical}
+
+The graphical installer is recommended for desktop users and will guide you
+through the installation.
+
+1.   In the "Welcome" screen, you can select the language of the Installer and
+     the installed system.
+
+     ::: {.tip}
+     Leaving the language as "American English" will help to be able to copy
+     and paste error messages in a search engine or issue report.
+     :::
+
+2.   Next you should choose your location to have the timezone set correctly.
+     You can actually click on the map!
+
+     ::: {.note}
+     The installer will use an online service to guess your location based on
+     your public IP address.
+     :::
+
+3.   Then you can select the keyboard layout. The default keyboard model should
+     work well with most desktop keyboards. If you have a special keyboard or
+     notebook, your model might be in the list. Select the language you are most
+     comfortable typing in.
+
+     ::: {.tip}
+     There are special keyboard layouts that are optimized to be ergonomic like
+     [NEO](https://www.neo-layout.org/),
+     [Dvorak](https://en.wikipedia.org/wiki/Dvorak_keyboard_layout) and
+     [Colemak](https://colemak.com/), in opposition to the classic QWERTY layout
+     which comes from typewriters and has not changed much since 1873.
+     You might want to check them out.
+     :::
+
```

Comment:
```suggestion
```
I don't think we should promote that here. If a beginner chooses one of these and doesn't actually know what the consequences are then you cannot log in after a reboot.

Reply:
You are right. That information might not be useful and rather distracting here.

Should the mention of NEO in the existing documentation also be removed?

Reply:
We can probably keep it if it is something interesting other distros lack.